### PR TITLE
Home: Design Updates.

### PR DESCRIPTION
**Pull request by shicholas**
Breaking this down like Stirpe's new homepage, http://stripe.com, we can
create 4 vertically-expanded sections with the copy in this PR.

For the second section, the one with a 5x2 grid of questions (in desktop
mode), I was envisioning that when a user presses a question there would
be text beneath that question explaining to him or her how we can help.

Stripe is a great exemplar website because they have great fonts and
colors, and use of icons throughout the page. Their page also has a lot
of breathing room - which is something you have commented on.

### CODE REVIEW

**Comment by nisarhassan12** [comment]
Thanks, @shicholas.

I quite like the initiative :100:  And it is indeed a lot better then what we currently have but there is still a lot of things that can be improved. 

I have heard this from a UX Expert Rob Hope that: "It's always a good idea to convince the user with as little as possible."

I would say in the Banner(Hero Area) It would be a lot better if we were to just have the "Headline" along with the mission statement with the CTA below maybe taking up  `100vh` (100% of the viewport width) with some Background (gradient, pattern or even a good background image As goog imagery helps build trust).

it is kind of a pattern that is used by most of the Brands out there.

image

image

image


After the Hero(Banner) we can have features sections each specifying a single service(What can we help with) in an un-cluttered direct manner with some good icons or imagery and so on.

>  I was envisioning that when a user presses a question there would
be text beneath that question explaining to him or her how we can help.

Thanks. Sorry If I'm unable to understand what you exactly meant. From what I know when things are hidden and shown only on a user action like "hover" or "clicking" a button there are a lot fewer chances that the contents will be viewed. I think it would be ok if we were to display them all without the need for the interaction with some breathing room and Big Typography i.e it's ok to do here as we don't have a lot of things going on the Home Page and the info is quite important. What do you say :) ?

**Comment by nisarhassan12** [comment]
Also currently we're missing some brilliant Testimonials. Testimonials are key to building trust. Like we can put some feedback from the people which "NeonLaw" have helped in the past? 

A good testimonial would outline What problem NeonLaw solved or helped them with or How it made their life better and so on.

Having Social proof icons also helps with building trust.

File: src/pages/index.mdx [hunk 1]
```diff
@@ -31,28 +31,82 @@ import { Seo } from '@components/seo';
...
-because we want to be your lawyer when you solve your money problems.
+Our goal is singular - your upward mobility. Join the thousands of people who
+have entrusted us with providing legal solutions to make their lives better.
 
-Once you have a handle on your finances, we can help you [buy a
-home](/practice-areas/real-estate), [start and manage a
-business](/practice-areas/small-business), and share your wealth with your friends
-and family with an [estate](/practice-areas/estate-planning).
+<Button variantColor="teal">
+  Schedule a Free Consultation Now
```
Comment (nisarhassan12) [inline src/pages/index.mdx:40]: The idea of having a single main Call to Action button is great :100:

**Comment by nisarhassan12** [comment]
@shicholas If you like then I can take on this PR and work on implementing a new better design for Landing(Home) Page?

**Comment by shicholas** [comment]
> @shicholas If you like then I can take on this PR and work on implementing a new better design for Landing(Home) Page?

yes please do, I know the style needs to change considerably.

**Comment by nisarhassan12** [comment]
I was trying to use the following technique described [here](https://snook.ca/archives/html_and_css/font-size-with-rem) which makes sizing with rems super easy.

image

but It didn't work as expected for a few elements. 
I have opened an issue for this https://github.com/chakra-ui/chakra-ui/issues/1190

**Comment by shicholas** [comment]
sounds good, I saw his response. Let's talk about it on Tuesday. I did a big re-org of the codebase too to support multiple websites - we can go over that then too.

**Comment by nisarhassan12** [comment]
@shicholas I was trying to use `activeClassName` prop but it does not seem to be working:

```jsx
<Box
  as={Link}
  cursor="pointer"
  margin="0 10px"
  to={link.route}
  className="nav-link"
  activeClassName="nav-link--active"
>
  {link.label}
</Box>
```

The cause of the issue is specified here: https://github.com/chakra-ui/chakra-ui/issues/796
upgrading to 1.0 would hopefully fix this.

**Comment by nisarhassan12** [comment]
> Also currently we're missing some brilliant Testimonials. Testimonials are key to building trust. Like we can put some feedback from the people which "NeonLaw" have helped in the past?
> 
> A good testimonial would outline What problem NeonLaw solved or helped them with or How it made their life better and so on.
> 
> Having Social proof icons also helps greatly with building trust.

@shicholas Can you help me the contents for the above? Thanks

**Comment by nisarhassan12** [comment]
closing this in favour of #454 because this one has a lot of conflicts.

**Pull request closed without merging**